webapp/channels/src/components/with_tooltip/index.tsx

```diff
@@ -83,6 +83,8 @@ interface Props {
    */
     onOpen?: () => void;
     children: ReactElement;
+
+    placement?: Placement;
```

Comment:
Can you rename this to forcedPlacement we don't want folks to use this except it only certain circumstances

Reply:
Done.